ntbs-service/wwwroot/css/site.css

```diff
@@ -54,3 +54,53 @@ body {
   height: 60px;
   line-height: 60px; /* Vertically center the text there */
 }
+
+.patient-layout-flexbox {
+    display: flex;
+}
+
+.contents-list {
+    width: 300px;
+    padding-right: 80px;
+}
+
+.contents-list-item {
+    list-style-type: none;
+    margin-bottom: 5px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+
+}
+
+.contents-list-item--current {
```

Comment:
They are coming from standard browser styling not bootstrap AFAIK and the pagination component from the NHS only uses <a> tags where we need buttons (which also means I probably can't use the taghelper markup. I'm not sure if this affects accessibility much but it does affect the inbuilt styling from the NHS package which is why I've needed this fix!

Reply:
The taghelper shouldn't affect accessibility, but it will make it easier to write these bits - if we can get them to agree!

I'm suggesting we remove the button styling altogether, rather than selectively by applying classes to buttons where we want it removed. So something along the lines of https://meyerweb.com/eric/tools/css/reset/ to our stylesheets.